docker/Dockerfile

```diff
@@ -147,7 +176,10 @@ COPY --from=production-stage / /
 COPY --from=backend-build /src/.venv /src/.venv
 # Fix virtualenv link to python binary
 RUN ln -sf "$(which python)" /src/.venv/bin/python
-ENV PATH="/src/.venv/bin:${PATH}"
+
+ARG SLIM_IMAGE=false
+ENV PATH="/src/.venv/bin:${PATH}" \
+    SLIM_IMAGE="${SLIM_IMAGE}"
```

Comment:
Does it provide any extra value if we determine whether the current image is slim or full, in Python, by checking if `assets/emulatorjs` or `assets/ruffle` are present?

That way, users could still use a Slim image, mount a local copy of Ruffle (for example, if they don't need EmulatorJS), and still be able to enable Ruffle emulation.